pubspec.yaml

```diff
@@ -64,9 +64,6 @@ dependencies:
   # flutter_dotenv
   flutter_dotenv: 5.0.2
 
-  # flutter_typeahead
-  flutter_typeahead: 3.1.1
```

Comment:
we don't need it anymore? weird. I think, we only add the customization in `tmail-flutter`, all the other base of `flutter_typeahead` we still use from the upstream lib

Reply:
Currently, we don't use it anymore